guides/source/developers/extensions/writing-extensions.html.md

```diff
@@ -0,0 +1,67 @@
+# Writing a New Solidus Extension
+
+Sometimes building a site powered by Solidus will require customization.
+Solidus extensions are Rails engines that allow you to extend or change the
+functionality of the Solidus platform.
+
+## Find existing extensions
+
+Before you write a new Solidus extension you should look for an existing
+extension that meets your needs. Solidus maintains an official list of
+[supported extensions](http://extensions.solidus.io), but you can also use
```

Comment:
Not all extensions are listed at http://extensions.solidus.io. We should also mention https://github.com/solidusio-contrib and that people can register their extension at https://github.com/solidusio/extensions.solidus.io via opening a PR